### PR TITLE
Hail in PyCharm

### PR DESCRIPTION
**Pull request by vladsaveliev**

### CODE REVIEW

File: code_editors.md [hunk 1]
```diff
@@ -45,3 +47,50 @@ when using conda.
...
+next to the field for `Project Interpreter`:
+
+![pycharm-interpreter](figures/pycharm-interpreter.png)
+
+Choose `Add` in the menu that pops up, and select `Existing environment`:
+
+![pycharm-interpreter-add](figures/pycharm-interpreter-add.png)
+
+Click on the ellipsis symbol next to the field for `Interpreter`. Enter
+the path to python interpreter, e.g. `/Users/vlad/miniconda/envs/hail-dev/bin/python`.
```
Comment (illusional) [inline code_editors.md:77]:
```suggestion
the path to python interpreter (you could get this by running `which python` in the conda environment), e.g. `/Users/vlad/miniconda/envs/hail-dev/bin/python`.
```

**Review by illusional — APPROVED** [review]
Made one suggestion, and maybe making the images slightly smaller. But looks good!

File: code_editors.md (hunk 1 above)
Comment (vladsaveliev) [reply code_editors.md]: Thank you! :)

**Comment by vladsaveliev** [comment]
@illusional, good point on the image size - updating!

**Pull request merged**